apps/inheritance_app/inheritance_auth_wallet.c

```diff
@@ -287,7 +287,7 @@ static bool auth_wallet_get_seed_entropy() {
 
 static bool auth_wallet_pair_card() {
   card_error_type_e status = single_card_pair_operation(
-      (char *)ui_text_tap_the_card, ui_text_place_card_below);
```

Comment:
This was the only usage for this text, we can remove it from `constants_text.c/h` as well